doc/source/reference/routines.other.rst

```diff
@@ -35,3 +36,20 @@ NumPy version comparison
    :toctree: generated/
 
    lib.NumpyVersion
+
+Utility
+-------
+
+.. autosummary::
+   :toctree: generated/
+
+   get_include
+   deprecate
+   deprecate_with_doc
+
+Matlab-like Functions
+---------------------
+.. autosummary::
+   :toctree: generated/
+
+   who
```

Comment:
I think you only need to touch the RST file here, there should be no need to duplicate the docstrings